mobile/library/swift/EnvoyCxxSwiftInterop/cxx_swift_interop.h

```diff
@@ -0,0 +1,43 @@
+#import "library/cc/bridge_utility.h"
+#import "library/cc/direct_response_testing.h"
+#import "library/cc/engine_builder.h"
+#import "library/common/data/utility.h"
+#import "library/common/extensions/filters/http/platform_bridge/c_types.h"
+#import "library/common/main_interface.h"
+#import "library/common/network/apple_platform_cert_verifier.h"
+#import "library/common/stats/utility.h"
+
+namespace Envoy {
+namespace CxxSwift {
+
+using StringVector = std::vector<std::string>;
+using StringPair = std::pair<std::string, std::string>;
+using StringPairVector = std::vector<StringPair>;
+using StringMap = absl::flat_hash_map<std::string, std::string>;
+using HeaderMatcherVector = std::vector<DirectResponseTesting::HeaderMatcher>;
+using BootstrapPtr = intptr_t;
+
+inline void string_map_set(StringMap& map, std::string key, std::string value) { map[key] = value; }
```

Comment:
I think we might want to `std::move()` these strings to avoid a copy (or at least push the copy up to the caller).